pkg/api/v1alpha1/oidcconfig.go

```diff
@@ -38,31 +41,35 @@ func getOIDCConfig(fileName string) (*OIDCConfig, error) {
 	return &config, nil
 }
 
-func validateOIDCConfig(config *OIDCConfig) error {
+func validateOIDCConfig(config *OIDCConfig) field.ErrorList {
+	var errs field.ErrorList
+
 	if config == nil {
 		return nil
 	}
 
 	if config.Spec.ClientId == "" {
-		return fmt.Errorf("OIDCConfig clientId is required")
+		errs = append(errs, field.Invalid(field.NewPath("spec", "OIDCConfig", "ClientId"), config.Spec.ClientId, "OIDCConfig clientId is required"))
```

Comment:
sorry to come back to this again
do you know if the standard is to capitalize the field names or not?
I guess we either capitalize all or none?

Reply:
I don't think there is a standard in place. I see that `spec` is usually all lowercase but the capitalization for the field names to follow vary across our files. It's also a weird case here because OIDC is capitalized in our config since it is an acronym